components/org.wso2.carbon.appmgt.mobile/src/main/java/org/wso2/carbon/appmgt/mobile/mdm/AppDataLoader.java

```diff
@@ -35,6 +35,7 @@
 public class AppDataLoader {
 
     private static final Log log = LogFactory.getLog(AppDataLoader.class);
+    private static final String iOSDefaultTransportProtocol = "https";
 
```

Comment:
Change the variable name to caps since its a best practice to follow that when dealing with constant

Reply:
This is fixed wso2/carbon-appmgt#711